rules/calendar-do-you-set-your-work-hours-in-your-calendars/rule.md

```diff
@@ -21,6 +21,22 @@ You should always set your working hours in your Outlook calendar so others know
 <!--endintro-->
 
 You can find this setting in File | Options | Calendar Options, it looks like below:
-![Choose your working hours](Outlook work hours.jpg)     Figure: Outlook Calendar options for working hours settings
+
+![Choose your working hours](Outlook work hours.jpg)     
+**Figure: Outlook Calendar options for working hours settings**
```

Comment:
The Figure should go inside the alt text (the text in the [] of the image) 
e.g.
![Figure: Outlook Calendar options for working hours settings](Outlook work hours.jpg)

Reply:
Same thing on line 30, 33, 36, 41